### ISSUE TITLE
Add private comments to issues

### ISSUE THREAD

**Issue by Janus451** [go-gitea/gitea#25703]
### Feature Description

I would really love to be able to create private comments in an issue that are only seen by certain users. At the moment every activity in an issue can be seen by everybody who has access to the issue. If e.g. a customer created an issue internal discussions about that issue have to take place in a different location. If it would be possible to create comments that can't be seen by everybody all discussions could be documented in one location.

**Comment by sh7dm** [comment]
Agreed. GitLab CE has a similar feature that limits access to only ones with reporter privileges. It's called "Internal note"
